templates/checkout/_partials/cart-detailed-product-line.tpl

```diff
@@ -23,166 +23,159 @@
   * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
   *}
 
-<div class="product-line-grid d-flex">
-  <div class="product-line-grid-left col-md-2 col-xs-4">
-    <span class="product-line-grid-image media-middle">
-      {if $product.default_image}
-        <img src="{$product.default_image.bySize.cart_default.url}" alt="{$product.name|escape:'quotes'}" loading="lazy">
-      {else}
-        <img src="{$urls.no_picture_image.bySize.cart_default.url}" loading="lazy" />
-      {/if}
-    </span>
-  </div>
-
-  <div class="product-line-grid-right col-md-10 col-xs-8">
-    <div class="product-line-top-content">
-      <div class="product-line-top">
-        <a class="product-line-title" href="{$product.url}" data-id_customization="{$product.id_customization|intval}">{$product.name}</a>
-
-        <div class="cart-line-actions">
-          <a
-              class                       = "remove-from-cart"
-              rel                         = "nofollow"
-              href                        = "{$product.remove_from_cart_url}"
-              data-link-action            = "delete-from-cart"
-              data-id-product             = "{$product.id_product|escape:'javascript'}"
-              data-id-product-attribute   = "{$product.id_product_attribute|escape:'javascript'}"
-              data-id-customization   	  = "{$product.id_customization|escape:'javascript'}"
-         >
-            {if empty($product.is_gift)}
-              <i class="material-icons float-start">delete</i>
+<div class="product-line-grid row">
+    <div class="product-line-grid-left col-md-2 col-4">
```

Comment:
IMO all templates should have indentation of 2 spaces. They are already packed and 4 spaces makes it harder to read.

Reply:
this temple has 2 space indentation. .editorconfig has this setting

Reply:
Maybe it has, but clearly isn't applied here. That's how raw file looked before:

![obraz](https://user-images.githubusercontent.com/5007456/144446499-032b3b81-dbee-4e86-b49d-bc6f9f9104a2.png)

and now made with this PR:

![obraz](https://user-images.githubusercontent.com/5007456/144446778-4613d0ab-1a80-43c9-8e3e-fa01fa770f4e.png)

Reply:
Yes, if you use VS code, change the amount of spaces to be used for indentation to 2.

Reply:
Should we add `tab_width` to the editorconfig? I think so